vignettes/custom-expectation.Rmd

```diff
@@ -12,9 +12,9 @@ library(testthat)
 knitr::opts_chunk$set(collapse = TRUE, comment = "#>")
 ```
 
-This vignette shows you how to create custom expectations that work identically to the built-in `expect_` functions.  Since these functions will need to be loaded when your package is loaded for testing, it is recommended that `expect_` functions be defined in `test-helpers.R` in your packages `R/` directory.
+This vignette shows you how to create custom expectations that work identically to the built-in `expect_` functions. Since these functions will need in order for your tests to work, we recommend putting them in an appropriately named helper function, i.e. `tests/testthat/helper-expectations.R`.
```

Comment:
I think "to be loaded" or something similar needs to come back.

In "putting them in an appropriately named helper **function**", do you mean **file**?